src/core/lib/security/credentials/credentials.h

```diff
@@ -130,11 +130,27 @@ struct grpc_channel_credentials
     return args;
   }
 
-  virtual int cmp(const grpc_channel_credentials* other) const = 0;
+  // Compares this grpc_channel_credentials object with \a other.
+  // If this method returns 0, it means that gRPC can treat the two channel
+  // credentials as effectively the same. This method is used to compare
+  // `grpc_channel_credentials` objects when they are present in channel_args.
+  // One important usage of this is when channel args are used in SubchannelKey,
+  // which leads to a useful property that allows subchannels to be reused when
+  // two different `grpc_channel_credentials` objects are used but they compare
+  // as equal (assuming request_metadata_creds and other properties on the
+  // subchannel_connector also match).
```

Comment:
I assume you mean security_connector, not subchannel_connector.

But more generally, I think this can just say "assuming other channel args also match".